content/user/admin_pages/menu_sections.md

```diff
@@ -29,10 +29,10 @@ index: false
 *   [Product Info](/user/admin_pages/configuration/configuration_productinfo/) — Basic options controlling elements to include on _Product Info_ page.
 *   [Layout Settings](/user/admin_pages/configuration/configuration_layoutsettings/) — Controls certain layout options.
 *   [Website Maintenance](/user/admin_pages/configuration/configuration_websitemaintenance/) — Controls how your store appears to customers when it is down for maintenance.
-*   [New Listing](/user/admin_pages/configuration/configuration_newlisting/) — How and where certain product listing information for new listings are displayed.
-*   [Featured Listing](/user/admin_pages/configuration/configuration_featuredlisting/) — How and where certain product listing information for featured listings are displayed.
-*   [All Listing](/user/admin_pages/configuration/configuration_alllisting/) — How and where certain product listing information for all listings are displayed.
-*   [Index Listing](/user/admin_pages/configuration/configuration_indexlisting/) — Whether certain items show on main page.
+*   [New Listing](/user/admin_pages/configuration/configuration_newlisting/) — How the New Products listing page should be laid out. 
+*   [Featured Listing](/user/admin_pages/configuration/configuration_featuredlisting/) — How the Featured Products listing page should be laid out.
+*   [All Listing](/user/admin_pages/configuration/configuration_alllisting/) — How the All Products listing page should be laid out.
```

Comment:
Given these are all the same, combining them into one page makes better sense IMO

Reply:
These pages are autogenerated from the configuration table so that would be a bit tricky.

Reply:
True.

I find that whole section of the docs very hard to use. 
Especially since in the left nav it creates a massive scrolling list of things I have to scroll past to find what I'm really looking for. Basically it makes the left nav pointless, so why do we even have it?

Reply:
If you want me to get rid of the left nav, just ask!  For some reason, I thought you were a fan so I didn't do it when I removed the right hand sidebar.

Reply:
No, that's not really my point. 

My point is that all the configuration-menu-entry stuff clutters the left nav. Would prefer if those entries could be excluded.

Reply:
ah ok - you can probably exclude some stuff from the left nav - not sure.